lib/administrate/fields/enum.rb

```diff
@@ -0,0 +1,23 @@
+require_relative "string"
+
+module Administrate
+  module Field
+    class Enum < Field::String
+      def self.search_predicate(attribute, term, options)
+        if options.keys.include?(term)
+          "#{attribute} = ?"
+        else
+          nil
+        end
+      end
+
+      def self.search_term(attribute, term, options)
```

Comment:
Unused method argument - `attribute`. If it's necessary, use `_` or `_attribute` as an argument name to indicate that it won't be used.